redis_to_sql.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+import argparse
+import os
+from datetime import datetime
+
+import redis
+from dotenv import load_dotenv
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from models import DailyHours
+import utilities
+
+load_dotenv("dev.env")
+
+mode = "development"
+
+# command line arguments
+parser = argparse.ArgumentParser(description="Migrate redis sets to mysql")
+parser.add_argument('time', help="The time of the redis set to migrate in format hh:mm:ss")
+parser.add_argument('--date', default=datetime.today().strftime('%Y-%m-%d'),
+                    help="The date of the redis set to migrate in format Y-m-d. Default is today's date")
+args = parser.parse_args()
+
+sorted_set_name = f"daily_{args.date} {args.time}"
+sorted_set_datetime = datetime.strptime(sorted_set_name, 'daily_%Y-%m-%d %H:%M:%S')
```

Comment:
nitpick...
Could you try to use the configured `datetime_format` like here?
https://github.com/Study-Together-Org/time_counter/blob/737246b54406c3b055f80906127220701e8dff6c/utilities.py#L219

Reply:
Yes added ✅